src/ecs/transform/systems.rs

```diff
@@ -209,4 +168,211 @@ mod tests {
         let transform: Transform = primitive.into();
         assert_eq!(primitive, transform.0);
     }
+
+    fn transform_world<'a, 'b>() -> (World, Dispatcher<'a, 'b>) {
+        let mut world = World::new();
+        world.register::<LocalTransform>();
+        world.register::<Transform>();
+        world.register::<Parent>();
+
+        let dispatcher = DispatcherBuilder::new()
+            .add(TransformSystem::new(), "amethyst/transform", &[])
+            .build();
+
+        (world, dispatcher)
+    }
+
+    fn together(transform: Transform, local: LocalTransform) -> [[f32; 4]; 4] {
+        (Matrix4::from(transform.0) * Matrix4::from(local.matrix())).into()
+    }
+
+    // Basic default LocalTransform -> Transform (Should just be identity)
+    #[test]
+    fn zeroed() {
+        let (mut world, mut dispatcher) = transform_world();
+
+        let mut transform = LocalTransform::default();
+        transform.translation = [0.0, 0.0, 0.0];
+        transform.rotation = [1.0, 0.0, 0.0, 0.0];
+
+        let e1 = world.create_entity()
+            .with(transform)
+            .with(Transform::default())
+            .build();
+
+        dispatcher.dispatch(&mut world.res);
```

Comment:
Why not just do `RunNow` on the system? Feels like unnecessary overhead to use a real `Dispatcher`.

Reply:
Was just easier to set up, overhead doesn't really matter that much in these tests anyways.

Reply:
How is it easier? It's more lines.

Reply:
Mainly because I know it by memory,.

Reply:
`system.run_now(&mut world.res)` ;)